iXBRLViewerPlugin/viewer/src/js/viewer.js

```diff
@@ -234,26 +234,44 @@ Viewer.prototype.contents = function() {
     return this._iframes.contents();
 }
 
-
-Viewer.prototype._selectAdjacentTag = function (offset) {
-    var elements = $(".ixbrl-element:not(.ixbrl-continuation)", this._contents);
-    var current = $(".ixbrl-selected:not(.ixbrl-continuation)", this._contents);
-    var next;
-    if (current.length == 1) {
-        next = elements.eq((elements.index(current.first()) + offset) % elements.length);
+// Move by offset (+1 or -1) through the tags in the document in document
+// order.
+//
+// Each element may have one or more tags associated with it, so we need to
+// move through the list of tags associated with the current element before
+// moving to the next/prev element
+//
+Viewer.prototype._selectAdjacentTag = function (offset, currentFact) {
```

Comment:
To be sure, the param is named `currentFact` (throughout), but I guess it's really `currentItem` since the inspector's current item could be a footnote, right?

Reply:
Yes, I'll rename it.